lora-phy/src/lib.rs

```diff
@@ -199,7 +199,7 @@ where
                 self.wait_for_irq().await?;
                 match self.radio_kind.process_irq_event(self.radio_mode, None, true).await {
                     Ok(Some(IrqState::Done | IrqState::PreambleReceived)) => {
-                        self.radio_mode = RadioMode::Standby;
+                        //self.radio_mode = RadioMode::Standby;
```

Comment:
Where do we manually switch `self.radio_mode = RadioMode::Standby;` after tx operation?

Reply:
both changes sneaked trough, the Standby after tx is another issue. Right now prepare_for_tx puts in transmit mode and ideally it should be possible to transmit multiple times with one setup but it is not possible as tx() goes to standby. Ill fix my PR later. Thanks for catching